app/assets/javascripts/tags.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Te pliki się tworzą automatycznie, bez mojej wiedzy. NIe usuwam, bo nie wiem (na razie) po co są.
  ----- Original Message ----- 
  From: Anna Ślimak 
  To: kraryk/test_blog 
  Cc: kraryk 
  Sent: Tuesday, January 26, 2016 3:06 PM
  Subject: Re: [test_blog] BŁąd przy wywoływaniu @post.tags (#3)


  In app/assets/javascripts/tags.js.coffee:

> @@ -0,0 +1,3 @@
> +# Place all the behaviors and hooks related to the matching controller here.
> +# All this logic will automatically be available in application.js.
> +# You can use CoffeeScript in this file: http://coffeescript.org/
Staraj sie nie dodawac pustych plikow

  —
  Reply to this email directly or view it on GitHub.